src/clients/jobManagerWrapper.ts

```diff
@@ -81,6 +82,7 @@ export class JobManagerWrapper extends JobManagerClient {
           parameters: {
             isNewTarget: true,
             targetFormat: data.targetFormat,
+            outputFormatStrategy: TileFormatStrategy.Mixed,
```

Comment:
This is already passed as `mixed` through `data` from `createPackageManager`.
I think you should either use from data or don't pass with data.

Reply:
fixed